# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/StructuredCloneTags.h
# Commit: ea637545c497
# Full Hash: ea637545c4972c106a7e2ed801cb514b9412ff62
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-03-06 04:41:40
# Regressor Bug: 1858958
# File Overlap Count: 1
# Description:
#   Bug 1858958 - AudioData: structured cloning. r=chunmin,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D190948
# ==============================================================================

diff -r fdc20c3b80d6 -r ea637545c497 dom/base/StructuredCloneTags.h
--- a/dom/base/StructuredCloneTags.h	Tue Mar 05 15:52:12 2024 +0000
+++ b/dom/base/StructuredCloneTags.h	Tue Mar 05 15:52:13 2024 +0000
@@ -157,6 +157,8 @@
 
   SCTAG_DOM_ENCODEDVIDEOCHUNK,
 
+  SCTAG_DOM_AUDIODATA,
+
   // IMPORTANT: If you plan to add an new IDB tag, it _must_ be add before the
   // "less stable" tags!
 };